src/components/standalone/monitoring/NetifyInformaticsContent.vue

```diff
@@ -0,0 +1,200 @@
+      }
+    })
+    .catch((exception: AxiosError) => {
+      errorSaving.value.notificationTitle = t('error.cannot_save_configuration')
+      errorSaving.value.notificationDescription = t(getAxiosErrorMessage(exception))
+    })
+    .finally(() => (saving.value = false))
+}
+
+function copyUuid() {
+  navigator.clipboard.writeText(form.value.uuid)
+  justCopied.value = true
+  setTimeout(() => {
+    justCopied.value = false
+  }, 3000)
+}
+</script>
+
+<template>
+  <div>
+    <FormLayout class="max-w-6xl">
+      <template #description>
+        <p>{{ t('standalone.netify_informatics.content_description') }}</p>
+        <NeLink href="https://portal.netify.ai/login" target="_blank">
+          <FontAwesomeIcon class="mr-2" :icon="faArrowUpRightFromSquare" />
+          {{ t('standalone.netify_informatics.open_netify_dashboard') }}
+        </NeLink>
+      </template>
+      <NeSkeleton v-if="loading" :lines="5" />
+      <NeInlineNotification
+        v-if="errorLoadingConfiguration.notificationTitle"
+        class="my-4"
+        kind="error"
+        :title="errorLoadingConfiguration.notificationTitle"
+        :description="errorLoadingConfiguration.notificationDescription"
+      />
+      <NeInlineNotification
+        v-if="errorSaving.notificationTitle"
+        class="my-4"
+        kind="error"
+        :title="errorSaving.notificationTitle"
+        :description="errorSaving.notificationDescription"
+      />
+      <div v-if="!isError && !loading" class="mb-8 flex flex-col gap-y-6">
+        <div class="relative flex items-end gap-2">
+          <NeTextInput
+            v-model="form.uuid"
+            disabled
+            :invalid-message="error.uuid"
+            :label="t('standalone.netify_informatics.uuid')"
+            ref="uuidRef"
+          >
+          </NeTextInput>
+          <NeTooltip v-if="justCopied" triggerEvent="mouseenter focus" placement="top-start">
+            <template #trigger>
+              <NeButton kind="secondary" size="md" @click="copyUuid" class="mb-px ml-2.5">
+                <template #prefix>
+                  <FontAwesomeIcon :icon="faCopy" class="h-4 w-4" aria-hidden="true" />
+                </template>
+                {{ t('standalone.netify_informatics.copy') }}
+              </NeButton>
+            </template>
+            <template #content>
+              {{ t('standalone.netify_informatics.copied_to_clipboard') }}
+            </template>
+          </NeTooltip>
+          <NeButton v-else kind="secondary" size="md" @click="copyUuid" class="mb-px ml-2.5">
+            <template #prefix>
+              <FontAwesomeIcon :icon="faCopy" class="h-4 w-4" aria-hidden="true" />
+            </template>
+            {{ t('standalone.netify_informatics.copy') }}
+          </NeButton>
+        </div>
+        <NeToggle
+          v-model="form.status"
+          :topLabel="t('standalone.netify_informatics.status')"
+          @update:model-value="handleSaveAutomaticToggle"
```

Comment:
This is dangerous, you're overriding the [vue default compilation](https://vuejs.org/guide/components/v-model.html#under-the-hood).

I'd suggest the usage of `watch` instead, if you used it before and the trigger was not working, could be cause of the missing [deep directive](https://vuejs.org/guide/essentials/watchers.html#deep-watchers), which if you simplify the `refs` this is not needed anymore.